src/lib/fauna-client.mjs

```diff
@@ -122,3 +128,28 @@ export const formatQueryResponse = (
     return faunaV10.formatQueryResponse(res, { raw, format, color });
   }
 };
+
+/**
+ * Formats a performance hint. If no hint is available, returns a default message. If
+ * the hint is malformed, returns the hint as is.
+ * @param {string} performanceHint - The performance hint
+ * @returns {string}
+ */
+export const formatPerformanceHint = (performanceHint) => {
```

Comment:
we currently just pass this the entire summary right? the summary can contain multiple performance hints

Reply:
This has been updated to support showing the summary in general as well as rendering multiple performance hints.

![Screenshot 2024-12-10 at 3 52 00 PM](https://github.com/user-attachments/assets/91f328d9-1ba6-4c7d-a714-3f53fcb03d3b)